lib/assets/tmpl-spec.jst

```diff
@@ -15,24 +17,24 @@ var assert = require('assert'),
     'font', 'ins', 'del', 'pre', 'address', 'dt',
     'q', 'i', 'b', 'u', 's', 'bdo', 'em'
     ]},
-    saveHtmlFile = function(filenames, actual) {
+    saveToFile = function(filenames, actual) {
         if (!Array.isArray(filenames)) {
             filenames = [filenames];
         }
-        var html = beautifyHtml(actual);
+
+        var data = typeof actual === 'string' ?
+            beautifyHtml(actual) :
+            '(' + stringifyObject(actual, { indent: '    '}) + ')'; 
```

Comment:
let's move `{ indent: '    '}` as a default value to an option and reuse everywhere (now it's copy/pasted 3 times)